plugins/shell/src/process/mod.rs

```diff
@@ -262,6 +262,31 @@ impl Command {
     ///     Ok(())
     /// });
     /// ```
+    ///
+    /// Depending on the command you spawn, it might output in a specific encoding, to parse the output lines in this case:
+    ///
+    /// ```rust,no_run
+    /// use tauri_plugin_shell::{process::{CommandEvent, Encoding}, ShellExt};
+    /// tauri::Builder::default()
+    ///   .setup(|app| {
+    ///     let handle = app.handle().clone();
+    ///     tauri::async_runtime::spawn(async move {
+    ///       let (mut rx, mut child) = handle.shell().command("cargo")
```

Comment:
`cargo` doesn't output in `windows-1252`, let's change it to maybe `some-program`, and the `tauri dev` args